CMakeLists.txt

```diff
@@ -152,17 +152,17 @@ if (CMAKE_BUILD_SHARED_LIBS)
 endif()
 
 install(FILES
-  "include/errors.hpp"
-  "include/entry_iterator.hpp"
-  "include/path.hpp"
-  "include/temp.hpp"
-  "include/pathie.hpp"
+  "include/pathie/errors.hpp"
+  "include/pathie/entry_iter`ator.hpp"
```

Comment:
The backtrick probably shouldn't be there.

Reply:
It definitely should not be. It is removed by e6917b8.